inst/rmarkdown/templates/oup/resources/template.tex

```diff
@@ -169,72 +223,129 @@
+
+$if(authors.corresponding_author)$\corresp[\fnsymbol{thisauthcorresp}]{Corresponding author. \href{mailto:$authors.email$}{\nolinkurl{$authors.email$}}}$endif$
+$if(authors.thanks)$\corresp[\fnsymbol{thisauththanks}]{$authors.thanks$}$endif$
+
 $endfor$
-%
-% Add \and if both authors and author
-$if(authors)$$if(author)$\and$endif$$endif$%
-%
-% Code for new (elsevier) style author field
+
 $for(author)$
-\name{$author.name$}
-$if(author.affiliation)$
+
+\author[%
+$for(author.affiliation)$\addrlblbycode{$author.affiliation$}$sep$,$endfor$%
+$if(author.corresponding_author)$$if(author.affiliation)$,$endif$\refstepcounter{correspcnt}\setcounter{thisauthcorresp}{\value{correspcnt}}\fnsymbol{thisauthcorresp}$endif$%
+$if(author.thanks)$$if(author.affiliation)$,$elseif(author.corresponding_author)$,$endif$\refstepcounter{correspcnt}\setcounter{thisauththanks}{\value{correspcnt}}\fnsymbol{thisauththanks}$endif$%
+$if(author.footnote)$$if(author.affiliation)$,$elseif(author.corresponding_author)$,$elseif(author.thanks)$,$endif$$for(author.footnote)$\corresplblbycode{$author.footnote$}$sep$,$endfor$$endif$%
+]{$author.name$}
+
 $for(author.affiliation)$
-\address{\Ref{ADR}{$author.affiliation$}}
+\addrbycode{$author.affiliation$}
 $endfor$
-$endif$%
-$if(author.email)$\email{\href{mailto:$author.email$}{$author.email$}}$endif$%
-$if(author.corresponding_author)$\thanks{Corresponding author; Email: \href{mailto:$author.email$}{$author.email$}}$endif$%
-$if(author.thanks)$\thanks{$author.thanks$}$endif$%
-$if(author.footnote)$
+
+$if(author.corresponding_author)$\corresp[\fnsymbol{thisauthcorresp}]{Corresponding author. \href{mailto:$author.email$}{\nolinkurl{$author.email$}}}$endif$
+$if(author.thanks)$\corresp[\fnsymbol{thisauththanks}]{$author.thanks$}$endif$
+
 $for(author.footnote)$
-\thanksbycode{$author.footnote$}
+\correspbycode{$author.footnote$}
 $endfor$
-$endif$%
-$sep$\and
+
 $endfor$
-%
-}
 
-\abstract{$abstract$}
+% Add author mark
+$if(authormark)$
+\authormark{$authormark$}
+$elseif(author)$
+\authormark{$for(author/first)$$it.name$$endfor$$if(author/rest/rest)$ et al.$else$$for(author/rest)$ \and $author.name$$endfor$$endif$}
+$elseif(authors)$
+\authormark{$for(authors/first)$$it.name$$endfor$$if(authors/rest/rest)$ et al.$else$$for(authors/rest)$ \and $authors.name$$endfor$$endif$}
+$endif$
+
+\received{Date}{0}{Year}
+\revised{Date}{0}{Year}
+\accepted{Date}{0}{Year}
 
-\date{$if(date)$$date$$else$\today$endif$}
+%\editor{Associate Editor: Name}
+
+\abstract{
+$for(abstract)$$abstract$$sep$\\
+$endfor$}
 
 \keywords{$for(keywords)$$keywords$$sep$; $endfor$}
 
-\maketitle
+$if(boxedtext)$
+\boxedtext{
+$boxedtext$}
+$elseif(boxedlist)$
+\boxedtext{
+\begin{itemize}
+$for(boxedtext)$
+\item $boxedtext$
+$endfor$
+\end{itemize}}
+$endif$
 
+\maketitle
 
 $for(include-before)$
 $include-before$
 $endfor$
 
 $body$
 
-$if(notes)$
-\begin{notes}
-$notes$
-\end{notes}
+%%%%%%%%%%%%%%
+
+%\begin{appendices}
+%\end{appendices}
```

Comment:
Should a solution for this needs to be found ? 

It is possible that this could be added at the end of the rmarkdown file this way 

````markdown

::: {.appendices latex=true}

# Section title of first appendix

blabla

## Subsection title of first appendix

and so ....
:::
````

Reply:
Thanks for the suggestion. Done.